config.js

```diff
@@ -13,17 +13,24 @@ System.config({
       "deps": [
         "jquery"
       ]
+    },
+    "showdown/dist/showdown.js": {
+      "format": "global"
     }
   },
   map: {
     "aurelia-after-attached-plugin": "github:aurelia-ui-toolkits/aurelia-after-attached-plugin@0.1.0",
     "aurelia-animator-css": "npm:aurelia-animator-css@1.0.1",
     "aurelia-bootstrapper": "npm:aurelia-bootstrapper@1.0.1",
+    "aurelia-event-aggregator": "npm:aurelia-event-aggregator@1.0.1",
```

Comment:
How is this behavior (If you do not, SystemJS will think aurelia-event-aggregator is an absolute url ) absent in the Javascript version (demo-kendo app)?